tests/integration_tests/md/md_config_threshold.yaml

```diff
@@ -1,17 +1,22 @@
 ensemble:
   name: nvt
-  dt: 0.1 # fs time step
+  dt: 0.2 # fs time step
```

Comment:
Why not use 0.5 fs - which is what we use most of the time for hydrogen-resolved MD? I tend to often try out default values first and this would be faster on first try.

Reply:
The model is barely trained, this is just so it can get some steps done. Most time in this test is spent on JIT compiling and training anyway.